README.md

```diff
@@ -1,7 +1,14 @@
-![Arkouda logo](pictures/arkouda_wide_marker1.png)
-
-# Arkouda (αρκούδα): NumPy-like arrays at massive scale backed by Chapel.
-## _NOTE_: Arkouda is under the MIT license.
+<p align="center">
+  <img src="pictures/arkouda_wide_marker1.png"
+/>
+<h2 align="center">Arkouda (αρκούδα) :bear:</br>Interactive Data Analytics at Supercomputing Scale</br>a Python API powered by Chapel</h2>
```

Comment:
Since you start another `p` tag on line 6, this one needs a closing tag. Or we can eliminate the 2nd `<p align="center">`. Either one is fine, but we should definitely have each tag open match a close.

Reply:
good catch!

Reply:
Removing the second `<p align="center">` removed the centering of the badges, but adding the closing tag worked like a charm! Thanks